src/components/TabFrame.vue

```diff
@@ -10,7 +10,7 @@
         v-if="isLoaded"
         @keyup.left="leftKeyPress"
         @keyup.right="rightKeyPress"
-        @keyup.space="spaceBarPress"
+        @keydown.space="spaceBarPress"
```

Comment:
This makes pressing space at the bottom of the screen open logs *and* jump down, which I don't think is the behavior people want